bedrock/firefox/views.py

```diff
@@ -526,3 +527,51 @@ def ios_testflight(request):
     return l10n_utils.render(request,
                              'firefox/testflight.html',
                              {'newsletter_form': newsletter_form})
+
+
+class FirefoxProductDesktopView(BlogPostsView):
+    blog_posts_limit = 3
+    blog_posts_template_variable = 'articles'
+    blog_slugs = 'firefox'
+
+    def get_template_names(request):
```

Comment:
Note that we should probably write some tests for these views (we can do this in a later commit)